src/riak_kv_index_hashtree.erl

```diff
@@ -81,7 +84,8 @@
                 path,
                 build_time,
                 trees,
-                use_2i = false :: boolean()}).
+                use_2i = false :: boolean(),
+                version :: undefined | non_neg_integer()}).
```

Comment:
Technically the `undefined |` is implicit in the type spec for any Erlang record field that doesn't include a default value. Maybe you already knew this and just added it for clarity, in which case I have no problem with that, but thought I'd point that out in case you'd rather cut down on the verbosity a little.